modules/autoskill.lua

```diff
@@ -141,15 +142,15 @@ executeSkill = function ()
 		-- _commandsForEachStageArray is a two-dimensional array with something like abc1jkl4.
 		local currentSkill = _commandsForEachStageArray[currentStage][currentTurn]
 
-		--[[Prevent exessive delay between skill clickings.
-			isFirstSkill = 1 means more delay, cause one has to wait for battle animation.
-			isFirstSkill = 0 means less delay.
-		--]]
-		local isFirstSkill = 1
 		if currentSkill ~= '0' and currentSkill ~= '#' then
 			for command in string.gmatch(currentSkill, ".") do
-				decodeSkill(command, isFirstSkill)
-				isFirstSkill = 0
+				-- Check that skills can be clicked
+				while not _battle.hasClickedAttack() and not _battle.isIdle() do end

```

Comment:
This line really improved AutoSkill performance.
My device takes around 15 seconds to process Order Change, so that old `wait()` call just wasn't cutting it. lol